api/test/common/proxy-implementations/proxy-tracer.test.ts

```diff
@@ -174,9 +174,9 @@ describe('ProxyTracer', () => {
       tracer.startSpan(name, options, ctx);
 
       // Assert the proxy tracer has the full API of the NoopTracer
-      assert.strictEqual(
-        NoopTracer.prototype.startSpan.length,
-        ProxyTracer.prototype.startSpan.length
+      assert.ok(
```

Comment:
seems function.length doesn't include arguments with default values

Reply:
Honestly, do we need this assertion?

Reply:
not really as a few lines below an actual call is done and verifed that arguments are forwarded.

Reply:
removed this assert.